src/JsonApiDotNetCore.MongoDb/Queries/Internal/QueryableBuilding/QueryableBuilder.cs

```diff
@@ -0,0 +1,83 @@
+using System;
+using System.Linq.Expressions;
+using JsonApiDotNetCore.Configuration;
+using JsonApiDotNetCore.Queries;
+using JsonApiDotNetCore.Queries.Expressions;
+using JsonApiDotNetCore.Queries.Internal.QueryableBuilding;
+using JsonApiDotNetCore.Resources;
+
+namespace JsonApiDotNetCore.MongoDb.Queries.Internal.QueryableBuilding
+{
+    /// <summary>
+    /// Drives conversion from <see cref="QueryLayer"/> into system <see cref="Expression"/> trees.
+    /// </summary>
+    public sealed class QueryableBuilder
```

Comment:
Instead of copy/pasting this class, you can set `layer.Projection = null` before calling the built-in version to achieve the same result.

Reply:
I tried this, it sounds like the right thing to do, but I run into the same problem I had when I was trying to use projections... In this case I can narrow it down to one parameter that's required by `QueryableBuilder`'s constructor which is `IModel model` with `IModel` being exposed by EFCore.

What can we do about this??